### ISSUE TITLE
This video is not available

### ISSUE THREAD

**Issue by virtualdreams** [anxdpanic/plugin.video.youtube#1005]
### Context

Please provide any relevant information about your setup

* Add-on Version: 7.0.9.2
* Kodi Version: 21,1
* Kodi GUI Language: German
* Operating System: LibreELEC
* Operating System Language:

------

### Expected Behavior

Please describe the behavior you are expecting.

Play the damn video. :)
Videos from my own channel plays fine.

------

### Current Behavior

What is the current behavior?

The video is not available.

------

### Steps to Reproduce

Please provide detailed steps for reproducing the issue.

1. Try to play https://www.youtube.com/watch?v=h-LfjLewYb4 or any othe video
2. 
3. 

------

### Log

Please include a complete [debug log](https://kodi.wiki/view/Log_file).

```

```2024-11-29 19:01:35.054 T:986   warning <general>: [plugin.video.youtube] Failed to retrieve video info - video_id: h-LfjLewYb4, client: android_youtube_tv, auth: True,
                                                   status: ERROR, reason: UNKNOWN
2024-11-29 19:01:35.188 T:986   warning <general>: [plugin.video.youtube] Failed to retrieve video info - video_id: h-LfjLewYb4, client: android_testsuite, auth: True,
                                                   status: UNPLAYABLE, reason: This video is not available
2024-11-29 19:01:35.189 T:986     error <general>: [plugin.video.youtube] yt_play.play_video - This video is not available:
                                                     File "/storage/.kodi/addons/plugin.video.youtube/resources/lib/plugin.py", line 16, in <module>
                                                       plugin_runner.run()
                                                     File "/storage/.kodi/addons/plugin.video.youtube/resources/lib/youtube_plugin/kodion/plugin_runner.py", line 62, in run
                                                       plugin.run(provider, context, focused=(current_uri == new_uri))
                                                     File "/storage/.kodi/addons/plugin.video.youtube/resources/lib/youtube_plugin/kodion/plugin/xbmc/xbmc_plugin.py", line 158, in run
                                                       result, options = provider.navigate(context)
                                                     File "/storage/.kodi/addons/plugin.video.youtube/resources/lib/youtube_plugin/kodion/abstract_provider.py", line 157, in navigate
                                                       result = handler(provider=self, context=context, re_match=re_match)
                                                     File "/storage/.kodi/addons/plugin.video.youtube/resources/lib/youtube_plugin/youtube/helper/yt_play.py", line 348, in process
                                                       media_item = _play_stream(provider, context)
                                                     File "/storage/.kodi/addons/plugin.video.youtube/resources/lib/youtube_plugin/youtube/helper/yt_play.py", line 75, in _play_stream
                                                       exc=exc, details=''.join(format_stack())

2024-11-29 19:01:35.197 T:884     error <general>: Playlist Player: skipping unplayable item: 0, path [plugin://plugin.video.youtube/play/?video_id=h-LfjLewYb4]


------

### Additional Information

Please provide any additional information that may be helpful.



------

**Comment by virtualdreams** [comment]
Installed 7.1.1.3 manually fixed the problem. Why is not available in the official repositories?

**Comment by MoojMidge** [comment]
> Why is not available in the official repositories?

Because it was only made available yesterday... It takes time for changes to be made, tested, released, reviewed and accepted. Be a little bit mre patient.

**Comment by mm1992** [comment]
> > Why is not available in the official repositories?
> 
> Because it was only made available yesterday... It takes time for changes to be made, tested, released, reviewed and accepted. Be a little bit mre patient.


But it is still odd. 
v7.1.1.1 has been never imported into the Kodi repository.
v7.1.1.3 has been released last Saturday which is 5 days ago as GH says itself.
Now you are probably working on v7.2.0 as there is a beta on MoojMidge/plugin.video.youtube.
However, in the meantime, on Kodi repository there still persists v7.0.9.2 which is useless as we discussed in the issues.
Could you possibly just push v7.1.1.3 into Kodi repo and then go back building 7.2.0.
Because there would always be some issues reported by the users, always a reason to create another newer version and there would be never a moment good enough to push it into the Kodi repo.
Or - if it's not you who apply changes in Kodi repo - who does it instead who I could ask?

**Comment by MoojMidge** [comment]
Time is a finite resource. Sorry to say but I really don't care about anyone's opinion on how I should spend my time, especially when the explanation as to why things take time are just ignored in order to push what is ultimately a self-serving agenda.

This is not a comment addressed to you specifically, but you are just the latest to come up with a plan that may be well intentioned but is not necessarily fully cognizant of all the factors at play.

There are at least 3 regressions that need to be investigated as to whether or not they should be fixed prior to publishing a release to the Kodi repository.

Should I just push a release and ignore those issues and the people who are impacted by them? What should I say to the next person in line who has an issue that just wants me to push another release to make their own life easier? Why is your case any different to theirs?

Changes and tests are done in my test repository to avoid polluting this main repository. Want me stop working on v7.2.0 or whatever future version may exist just to push a new release out? Then nothing will happen with any further releases at all, because that is where all the fixes that go into a new release are developed.

If someone wants releases to be pushed out faster then they need to contribute by testing and providing relevant feedback. The issues that were addressed in v7.1.1.3 were going on for weeks and only 2 people bothered to provide a debug log to actually help resolve the problem.

Almost all the changes in v7.2.0 were done while people raised issues about v7.0.9.2 and v7.1.1.1 but chose not to provide any useful information to actually fix the issue.

A v7.1.1.3 release was made available here and now it needs to be tested properly. This takes time. End of story.
